.github/workflows/master_push_and_pull_request_workflow.yml

```diff
@@ -0,0 +1,26 @@
+# The workflow to check master after push or pull request.
+name: Master and pull request checks
```

Comment:
As the default branch has been renamed from master to main already in this repository, this name and filename of this workflow file should reflect that.

Reply:
good catch, thanks.
Renamed + also renamed a file